client/src/pages/GetInvolvedPage/components/VolunteerForm/VolunteerForm.jsx

```diff
@@ -125,18 +115,18 @@ const VolunteerForm = () => {
 						className="volunteer-form-textarea"
 					/>
 				</div>
-				<button
-					type="submit"
-					disabled={isSubmitting}
-					className="volunteer-form-submit"
-				>
-					{isSubmitting ? (
-						<div className="form-submit-loading-icon"></div>
-					) : (
-						"SUBMIT"
-					)}
-					{/* later: transition the loader into a tick after success */}
-				</button>
+				<div className="volunteer-form-submit-container">
+					<span className="volunteer-form-submit-result">{result}</span>
+					<button
+						type="submit"
+						disabled={isSubmitting}
+						className={`volunteer-form-submit ${
+							isSubmitting ? "disabled" : ""
+						}`}
+					>
+						{isSubmitting ? "Submitting..." : "SUBMIT"}
```

Comment:
```suggestion
						{isSubmitting ? "Submitting" : "Submit"}
```
1. keep consistent the case - upper or camel
2. the button will expand/contract in this way - do we want this(?)